src/clean_macro_data.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import os
+import re
+import argparse
+
+import yaml
+import pandas as pd
+import boto3
+
+from helpers import gen_helpers as gen_h
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+def run_ipo(args):
+    """Loads config and cleans the companies dataset. Uses cleaned data to
+    generate new features.
+    Args:
+        args: From argparse, should contain args.config and optionally, args.save
+            args.config (str): Path to yaml file with load_data as a top level
+            key containing relevant configurations
+            args.save (str): If given, resulting dataframe will be saved to this
+            location.
+    Returns: None
+    """
+    with open(args.config, "r") as f:
+        config = yaml.load(f)
+
+    df = gen_h.read_data(args.input_file)
+    df.columns = ['year','no_ipos']
+
+    df.to_csv(args.save)
```

Comment:
This file seems like a lot of overhead for just reading, renaming the two columns, and saving

Reply:
Yes but its because only IPO is included right now. Ideally there would be more macro factors and data that would be merged together for easy joining.